assignments/arrays.js

```diff
@@ -63,34 +63,75 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+function getInventoryCar(carId) {
+  for (i = 0; i < inventory.length; i++){
+    if (inventory[i].id === carId){
+      return inventory[i];
+    }
+  }
 
+  return "car ID: " + carId + " not found";
+}
+console.log(`Car 33 is a ` + getInventoryCar(33).car_year + ' ' + getInventoryCar(33).car_make + ' ' + getInventoryCar(33).car_model);
```

Comment:
- Looks like your using back-ticks here, might as well use the template literal syntax 
`Car 33 is a ${getInventoryCar(33).car_year} ${getInventoryCar(33).car_make} ${getInventoryCar(33).car_model} .` (make sure you keep it wrapped in back-ticks.